x-pack/plugins/cases/public/common/translations.ts

```diff
@@ -108,6 +115,10 @@ export const TAGS = i18n.translate('xpack.cases.caseView.tags', {
   defaultMessage: 'Tags',
 });
 
+export const CASE_TYPE = i18n.translate('xpack.cases.caseView.caseType', {
+  defaultMessage: 'Case Type',
```

Comment:
I found it a bit confusing because internally a case has an attribute called type that is different from this. Maybe we should change to something like `Solution` or similar. @paulewing what do you think?

Reply:
@cnasikas `Solution` or `Type` would be ok with me. I think we still need to refactor how we're naming things internally in the saved object (we still need to remove the case connector stuff which would remove that `type` field internally). I agree that it's confusing internally. For the user `Type` could make sense, I think `Solution` would work as well.

Reply:
changed to Solution